runners/direct-java/src/main/java/org/apache/beam/runners/direct/ParDoEvaluatorFactory.java

```diff
@@ -87,7 +87,7 @@ public void cleanup() throws Exception {
               stepContext,
               application,
               application.getInput().getWindowingStrategy(),
-              fnManager.get(),
+              fnManager.get().getFn(),
```

Comment:
@kennknowles created https://github.com/apache/incubator-beam/pull/1428. It is easier for me to rebase this commit after that.